tests/openvino/test_modeling.py

```diff
@@ -1934,6 +1934,39 @@ def test_compare_to_transformers(self, model_arch):
             f"generation config : {gen_config}, transformers output {transformers_outputs}, ov_model output {ov_outputs}",
         )
 
+        # previous run was with legacy processing, one more run with features concatenation on preprocessing level
+        if (
```

Comment:
Above there are quite a lot of manipulations done to `ov_model` and `transformers_model`. Won't they interfere with the "new" inference here? Maybe better to have two separate test methods for legacy and non-legacy inference?

Reply:
problem that we can not trust warnings in transformers :) in 4.45 transformers raises warning about legacy processing says that it will be removed in 4.46 (not the first time when some decisions about changing interface delayed e.g. using Cache class instead tuples for past_key_values promises to remove tuples support in 4.43 and still here), but it still here and by default preprocessing initialized as legacy if additional parameters are not provided, so I have no info when  new processing will be by default and when legacy gone, that is why I'm trying to be ready to this moment

Reply:
moved in separated test